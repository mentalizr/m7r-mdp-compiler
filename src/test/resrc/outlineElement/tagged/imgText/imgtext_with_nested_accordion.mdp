@img-text[alt="relaxing picture"](sun_400x250-1.jpg)
    This is a simple paragraph

    Some further Text as a paragraph:

    // TODO You need to do something

    @accordion[id="myId"]
    --- Tip
        Tip: Keep on hacking!

#### Use your power