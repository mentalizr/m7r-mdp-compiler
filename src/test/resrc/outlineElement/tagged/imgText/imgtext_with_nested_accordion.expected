<div class="row" style="margin-bottom: 1.0em; margin-top: 1.0em">
    <div class="col-xs-12 col-sm-5 col-md-5 col-lg-5">
        <img src="media/sun_400x250-1.jpg" class="img-fluid" style="width: 100%" alt="relaxing picture">
    </div>
    <div class="col-xs-12 col-sm-7 col-md-7 col-lg-7">
        <p>This is a simple paragraph</p>
        <p>Some further Text as a paragraph:</p>
        <div class="accordion mt-3 mb-3" id="myId">
            <div class="card">
                <div class="card-header" id="myId_heading_0">
                    <h5 class="mb-0">
                        <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#myId_collapse_0" aria-expanded="true" aria-controls="myId_collapse_0">
                            Tip
                        </button>
                    </h5>
                </div>
                <div id="myId_collapse_0" class="collapse" aria-labelledby="myId_heading_0" data-parent="#myId">
                    <div class="card-body">
                            Tip: Keep on hacking!
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>